infrastructure/src/main/java/org/corfudb/infrastructure/BatchProcessor.java

```diff
@@ -193,9 +194,14 @@ private void processor() {
                             default:
                                 log.warn("Unknown BatchWriterOperation {}", currOp);
                         }
-                    } catch (Exception e) {
-                        log.error("Stream log error. Batch [queue size={}]. StreamLog: [trim mark: {}, tails: {}].",
-                                operationsQueue.size(), streamLog.getTrimMark(), streamLog.getAllTails(), e);
+                    } catch (OverwriteException ex) {
+                        log.trace("OverwriteException. Batch [queue size={}]. StreamLog: [trim mark: {}].",
```

Comment:
@xnull We do want to print a one-liner for OverwriteExceptions too. With this, it's difficult to know there were overwrite exceptions.

Reply:
fixed